src/themes/position-extend.scss

```diff
@@ -0,0 +1,6 @@
+.position-true-center{
+  position:absolute;
+  left:50%;
+  top:50%;
+  transform:translate(-50%,-50%);
+}
```

Comment:
Please merge the remote master branch,
and you will see the class ```.ab-center-by-transform``` in src/theme/centering.scss which do the same thing here.